testnet/bitcoin-neon-controller/src/main.rs

```diff
@@ -47,19 +47,31 @@ async fn main() -> http_types::Result<()> {
             Err(err) => err.duration(),
         }.as_secs() as u64;
 
-        let time_since_genesis = now - config.neon.genesis_timestamp;
-        
+        let genesis_timestamp = match std::env::var("NEON_GENESIS_TIMESTAMP") {
+            Ok(val) =>  match val.parse::<u64>() {
+                Ok(val) => val,
+                _ => config.neon.genesis_timestamp,
+            },
+            _ => config.neon.genesis_timestamp
```

Comment:
Would it make sense to default to something like `now` for the genesis_timestamp? That way this controller could be used in local setups a little more easily.

Reply:
Seconded.  I hit this problem while testing.  Having to set a timestamp in the config file every time I wanted to spin up a fresh test environment was annoying.

Reply:
Fine by me, we'll just have to be cautious on the master node, and ensure that the env var is set (which should be fine with our move to k8s)

Reply:
I'm a bit ambivalent about the approach of being destructive and resetting by default, how do you guys feel about https://github.com/blockstack/stacks-blockchain/pull/1577/commits/f2dedf7dbc18e138d0cc1aadfc1c64c9b3cc2f34?

Reply:
That works for me